frontend/src/FullPageRoutes/Profile.tsx

```diff
@@ -33,93 +34,145 @@ const useStyles = makeStyles({
   },
 })
 
+function parseBirthday(birth_date: Nullable<Date>): Nullable<Date> {
+  let birthday = null
+  if (birth_date !== null) {
+    birthday = new Date(birth_date)
+    // Typescript removes a day.
+    birthday.setDate(birthday.getDate() + 1)
+  }
+
+  return birthday
+}
+
```

Comment:
I'm a bit confused why we need to add 1 to the date here. Both the javascript Date and the python datetime.date appear to handle this correctly i.e. the date starts at 1. What was the problem you were having by not adding 1?

Reply:
See discord